qemu/tests/netkvm_cpu_mapping.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_net
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Get cpu mapping info from traceview session, and check
+    whether it matches the vectors count.
+
+    :params test: the test object
+    :params params: the test params
+    :params env: test environment
+
+    """
+
+    # boot the vm with the  queues
+    queues = int(params["queues"])
+    error_context.context("Boot the guest with queues = %s" % queues,
+                          logging.info)
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    nic = vm.virtnet[0]
+    nic_vectors = int(nic['vectors']) if nic['vectors'] else (2 * queues + 2)
```

Comment:
Is `vectors` optional? or could it be `''` or `'0'`?

Reply:
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/qemu_vm.py#L1843-L1849